src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -377,12 +377,27 @@ std::unique_ptr<RenderTree> RenderOrchestrator::createRenderTree(const UpdatePar
     }
     // Symbol placement.
     bool symbolBucketsChanged = false;
-    {
-        if (!isMapModeContinuous) {
-            // TODO: Think about right way for symbol index to handle still rendering
-            crossTileSymbolIndex.reset();
+    if (!isMapModeContinuous) {
```

Comment:
`if (!isMapModeContinuous) {` this is the most probable path, maybe move tile and static mode handling to `else` branch.